app/views/kalibro_ranges/new.html.erb

```diff
@@ -1,4 +1,4 @@
-<h1>New Range</h1>
+<h1><%= "#{t('new.male')} #{KalibroRange.model_name.human}" %></h1>
```

Comment:
We are going to discuss this soon. I do agree with you but fixing this will be a history of its own.